files/build_templates/manifest.json.j2

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.0.0",
+    "package": {
+        "version": "{{ version }}",
+        "base-os-contraint": "^{{ base_os_ver.split('.')[0] }}.0.0",
+        "depends": {{ depends.split()|json if depends is defined else []}},
+        "name": "{{ package_name }}"
+    },
+    "service": {
+        "name": "{{ name }}",
+        "requires": {{ requires.split()|json if requires is defined else [] }},
+        "after": {{ after.split()|json if after is defined else [] }},
+        "before": {{ before.split()|json if before is defined else [] }},
+        "dependent-of": {{ dependent_of.split()|json if dependent_of is defined else [] }},
+        "asic-service": {{ asic_service }},
```

Comment:
Guess, these could be optional and in absence, the value will be empty string. Is that correct ?
[SB] No, please refer to HLD https://github.com/stepanblyschak/SONiC/blob/sonic-app-ext-3/doc/sonic-application-extention/sonic-application-extention-hld.md#multi-asic.
The default is not an empty stirng and this field is not of type string.


I see, name is quoted "{{ name }}", but here it is not.
Is that because, asic-service is made of alphanumeric values only ?
[SB] Please refer to HLD https://github.com/stepanblyschak/SONiC/blob/sonic-app-ext-3/doc/sonic-application-extention/sonic-application-extention-hld.md#multi-asic.
Or could we safely put double quotes everywhere ?
[SB] No, as it is not always a stirng.